open-api/rest-catalog-open-api.yaml

```diff
@@ -1966,6 +2026,15 @@ components:
               "GET /v1/{prefix}/namespaces/{namespace}/tables/{table}",
               "GET /v1/{prefix}/namespaces/{namespace}/views/{view}"
             ]
+        idempotencyKeyLifetime:
+          type: string
+          format: duration
+          description: >
+            Minimum retention window for Idempotency-Key tokens (ISO-8601 duration,
```

Comment:
"Minimum retention" is not very useful from the client's perspective, IMHO. The client needs to decide when if can reuse a previous key for retries, but "minimum retention" sounds like something for the server.

How about `Maximum time period when the client can reuse a previous Idempotency-Key value. It is measured from the first request submission time using the key value to the submission time of the last retry (according to the client's clock). Servers should include some grace period into this time window to compensate for network delays and clock drift. Presence [...]` ?

Reply:
Thanks, I reworded based on your suggestion.